src/graphql/faas/mod.rs

```diff
@@ -28,8 +55,26 @@ impl FaasQueries {
     #[graphql(deprecation = "This is a test API and will be removed.")]
     #[instrument(skip(self))]
     async fn unix_to_clinks(&self, time: u64) -> Option<u64> {
-        Some(time)
-            .filter(|v| *v >= CLINK_OFFSET)
-            .map(|v| v - CLINK_OFFSET)
+        info!("[UnixToClinks] Processing Unix: {time}");
+
+        let res: Option<reqwest::Response> = reqwest::get(format!(
+            "https://ad-services.fnal.gov/faas/unix/{}",
+            time
+        ))
+        .await
+        .ok();
+
+        if let Some(resp) = res {
+            match resp.json::<ClinksUnix>().await {
+                Ok(clunx) => Some(clunx.clinks),
+                Err(er) => {
+                    info!("[UnixToClinks] Error: {er}");
+                    Some(0)
+                }
+            }
+        } else {
+            info!("[UnixToClinks] Response was not received");
```

Comment:
The "[UnixToClinks]" is redundant here, too, since line 55 instruments the whole function.